src/cljs/nr/gameboard/log.cljs

```diff
@@ -88,19 +77,23 @@
   "Matches if all characters in input appear in target in order.
   Score is sum of matched indices, lower is a better match"
   [input target]
-  (loop [curr-input (first input)
-         rest-input (rest input)
-         target-index (string/index-of target curr-input 0)
-         score target-index]
-    (when target-index
-      (if (not (seq rest-input))
-        score
-        (let [next-index (string/index-of target (first rest-input) (inc target-index))]
-          (recur
-            (first rest-input)
-            (rest rest-input)
-            next-index
-            (+ score (or next-index 0))))))))
+  ;; scoring is case insensitive
+  ;; TODO - scoring for similar letters (e.g. Şifr =~ sifr, 1337 =~ leet)
```

Comment:
I think we can use java.text.normalizer to deal with non-latin characters. No clue what the smart way to do the numbers is though.

Reply:
oh wait, **java**.text.normalizer? We're in ClojureScript land here 😅

Reply:
there is a [js equivalent](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/normalize) though which I think I can make work